README.md

```diff
@@ -0,0 +1,77 @@
+# DevOps Kathmandu Connect 2025
+
+[![Hugo Site](https://img.shields.io/badge/Built%20with-Hugo-ff4088)](https://gohugo.io/)
+[![License](https://img.shields.io/badge/License-MIT-blue.svg)](LICENSE)
+
+> Official website for DevOps Kathmandu Connect 2025 conference
+
+📅 **Event Website:** [https://devops-kathmandu.github.io/connect/](https://devops-kathmandu.github.io/connect/)  
+ℹ️ **About the Event:** [https://devops-kathmandu.github.io/connect/about-event/](https://devops-kathmandu.github.io/connect/about-event/)
+
+
+## Getting Started
+
+### Prerequisites
+- Hugo
+- Git
+
+<br>
+
+### Setup environment
+- Clone the repository
+
+```sh
+git clone https://github.com/devops-kathmandu/connect.git
+cd connect
+```
+
+- Setup theme
+
+```
+cd themes && git clone https://github.com/DevOps-Kathmandu/hugo-theme-event.git event
+cd ..
+```
+
+> Updating themes
+
+```
+# When theme updates are available
+cd themes/event
+git pull origin main
+cd ../..
+```
```

Comment:
Theme: `hugo-theme-event` is submodule, we could just use git submodule commands for these also.
 
 To clone the repo directly with submodules:
 ```
 git clone https://github.com/devops-kathmandu/connect.git --recurse-submodules
 ```
 
 Or if you cloned repo without submodules already:
```
git submodule update --init --recursive
```
 
 #### and to pull latest theme updates on remote (if needed)
 ```bash
 # check status of submodules
 git submodule status
 
#pull upstream updates
 git submodule update --remote --recursive
  ```
  
 If you need to commit the updated submodule reference:
```
git add .
git commit -m "Updated submodules to latest version"
git push
```